experiment/autobumper/bumper/bumper.go

```diff
@@ -261,3 +536,38 @@ func generatePRBody(images map[string]string, assignment string) string {
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
 	return prowSummary + "\n\n" + testImagesSummary + "\n\n" + assignment + "\n"
 }
+
+func getAssignment(oncallAddress string) string {
+	if oncallAddress == "" {
+		return ""
+	}
+	var oncaller string
+	var err error
+	if oncaller, err = getOncaller(oncallAddress); err == nil {
+		if oncaller != "" {
+			return "/cc @" + oncaller
+		}
+		return "Nobody is currently oncall, so falling back to Blunderbuss."
+	}
+	return fmt.Sprintf("An error occurred while finding an assignee: `%s`.\nFalling back to Blunderbuss.", err)
+}
+
+func getOncaller(oncallAddress string) (string, error) {
```

Comment:
This is only ever called once, are we sure we need it?

Reply:
Combined it with the `getAssignment` function.